examples/BlueMarbleTimeSeries.js

```diff
@@ -23,10 +23,13 @@ requirejs(['./WorldWindShim',
               LayerManager) {
         "use strict";
 
+        // Tell WorldWind to log only warnings.
         WorldWind.Logger.setLoggingLevel(WorldWind.Logger.LEVEL_WARNING);
 
+        // Create the WorldWindow.
         var wwd = new WorldWind.WorldWindow("canvasOne");
 
+        // Create and add layers to the WorldWindow.
```

Comment:
This example focuses in a single animated imagery layer, so layer creation and addition is handled differently. I suggest not straight replacing and copying the comments present in the other examples, but complementing the current comments with our usual boilerplate.